icu4c/source/python/icutools/databuilder/filtration_schema.json

```diff
@@ -38,7 +38,7 @@
                         "type": "array",
                         "items": {
                             "type": "string",
-                            "pattern": "^[+-]/((\\w+|\\*)(/\\w+|/\\*)*)?$"
+                            "pattern": "^[+-]/(([\\w%]+|\\*)(/[\\w%]+|/\\*)*)?$"
```

Comment:
This is fine for % but I think genrb allows anything that is an "invariant character", possibly minus spaces and quotation marks (I would have to look at the parser code).

How about just `"^[+-]/\\s$"` and let the genrb parser deal with key validity?